src/libstd/fs.rs

```diff
@@ -1536,6 +1531,28 @@ mod tests {
         TempDir(ret)
     }
 
+    // Several test fail on windows if the user does not have permission to create symlinks (the
+    // `SeCreateSymbolicLinkPrivilege`). Instead of disabling these test on Windows, use this
+    // function to test whether we have permission, and return otherwise. This way, we still don't
+    // run these tests most of the time, but at least we do if the user has the right permissions.
```

Comment:
Could you format this comment to 80 chars in width? (surrounding style of the module)